guides/doc-Configuring_Load_Balancer/topics/Registering_Clients.adoc

```diff
@@ -23,9 +23,11 @@ You must complete the registration procedure for each client.
 
 .Prerequisite
 
-Ensure that you install the bootstrap script on the client and change the script's file permissions to executable.
+* Ensure that you install the bootstrap script on the client and change the script's file permissions to executable.
```

Comment:
Is the bootstrap script still officially supported? I think upstream now only relies on the "host registration" feature.

Reply:
It is in our downstream.
Do you still support bootstrap and katello-ca-consumer in orcharhino?

Reply:
Yes, orcharhino still supports `bootstrap.py`.